tasks/idea/index.js

```diff
@@ -70,11 +70,7 @@ function createWorkspaceXml(lernaPackages, rootPackage, mochaConfigurations, log
   const nodePath = execSync('which node')
     .toString()
     .replace('\n', '')
-  const mochaPackage = join(
-    relative(rootPackage.location, lernaPackages[0].location),
-    'node_modules',
-    'mocha'
-  )
+  const mochaPackage = join('node_modules', 'mocha')
```

Comment:
yeah, this is not generic but tuned to our way of doing it (as some other stuff in it). I would rather move this to more generalization rather than tuning to our way. Maybe we could have couple of strategies (require.resolve, current shitty + expose override). I guess currently it's not a pressing matter so I will add it.